gtfs-netex-test/netexio/dbaccess.py

```diff
@@ -490,9 +490,12 @@ def insert_database(con, classes, f=None, cursor=False):
                 try:
                     cur.execute(sql_insert_object, (id, version, order, xml,))
                 except:
+                    print("affected element")
                     print(xml)
-                    raise
-                    pass
+                    #TOTO better fix for PassenderStopAssignments: We assume that they are the same. In reality we would need to check
+                    if not localname =="PassengerStopAssignment":
```

Comment:
What happened here?

Reply:
But if it has an order, than it should be in the database unique anyway. I want to know what breaks.

Reply:
You mean I should undo it, let the thing fail again with some data?
Then you can analyse the file. Is this the way to go?